core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7349RelocationReasonTest.java

```diff
@@ -57,6 +58,18 @@ public void testit()
         verifier.executeGoal( "verify" );
         verifier.resetStreams();
         verifier.verifyErrorFreeLog();
-        verifier.verifyTextInLog( "Test relocation reason" );
+        try {
+            verifier.verifyTextInLog("Test relocation reason");
+        } catch (VerificationException e) {
```

Comment:
Does duplicating the output make sense? The system will print a lot already and the log file can always be inspected by a dev.

Reply:
> Does duplicating the output make sense? The system will print a lot already and the log file can always be inspected by a dev.

Well, that's exactly because I did not find how to do it that I added the output.  The plan is to remove it when the test is fixed, but it's working locally and I currently have no idea why it fails.  The relocation warning is not displayed at all for an unknown reason.

Reply:
See https://github.com/apache/maven/runs/4838967658?check_suite_focus=true#step:7:5880